numpy/linalg/umath_linalg.c.src

```diff
@@ -3154,7 +3179,7 @@ GUFUNC_DESCRIPTOR_t gufunc_descriptors [] = {
     {
         "svd_m_f",
         "(m,n)->(m,m),(m),(n,n)",
-        "svd when m>=n",
+        "svd when m<=n",
```

Comment:
Copy and paste error. The rule is that the single letter in `()` is the minimum of `m` and `n`